index.js

```diff
@@ -56,11 +47,13 @@ async function main() {
       );
       return;
     }
-    const latestTag = await getLatestTagOnCurrentBranch();
-    const newTag = incrementTag(latestTag, releaseType);
-    await createNewTag(newTag);
-    await pushTags();
+
+    await exec(`git checkout ${process.env.GITHUB_BASE_REF}`);
```

Comment:
Can we add a comment here about what this is doing? Something like this maybe?
```suggestion
	// Checking out the target branch of the pull request.
    // GITHUB_BASE_REF is the base_ref or target branch of the pull request.
    // This property is only available when the event that triggers a workflow 
    // run is a pull_request.
    await exec(`git checkout ${process.env.GITHUB_BASE_REF}`);
```